pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types_conversion.go

```diff
@@ -0,0 +1,4 @@
+package v1beta1
+
+// Hub marks this type as a conversion hub.
+func (*PostgresCluster) Hub() {}
```

Comment:
❓ Hub sounds like webhook. Is this file used?

Reply:
No, it's removed in my next local commit